gramhopper/triggers/filter_triggers.py

```diff
@@ -33,32 +34,26 @@ def __init__(self, lang):
         super().__init__(Filters.language(lang))
 
 
+class _MessageTypeFilterBasedTrigger(FilterBasedTrigger):
+    def __init__(self, message_type=None):
```

Comment:
Why default to `None`? And if defaulting to `None`, maybe we want to check and do something if it's really `None`...

Reply:
Shouldn't be None. Removed the default value.